lib/LightsController/LightsController.cpp

```diff
@@ -230,7 +230,11 @@ void LightsControllerClass::ToggleLightState(Lights byLight, LightStates byLight
    {
       LightConfig.iColor = RgbColor(0);
    }
-   _LightsStrip->SetPixelColor(LightConfig.iPixelNumber, LightConfig.iColor);
+   for(int i=0; i<LIGHTSCHAINS; i++)
+      {
```

Comment:
Could you fix the indentation of the for loop please?

Reply:
That is not what I meant :) the use of `int i` was fine, but the opening and closing brace (`{`) are indented, while they should not be.
So instead of:
```
   for(int i=0; i<LIGHTSCHAINS; i++)
      {
      ...
      }
```
It should be:
```
   for(int i=0; i<LIGHTSCHAINS; i++)
   {
      ...
   }
```